### ISSUE TITLE
Improve documentation on how to position content on image markers

### ISSUE THREAD

**Issue by JamesBotterill** [AR-js-org/AR.js-Docs#1]
I know this is going to be potentially resolved in an update down the line, but right now it's hard to understand how to position content correctly on an image marker. 

It would therefore be good to get this explained on how it currently works.

**Comment by nicolocarpignoli** [comment]
it's actually empiric. it depends on sizes of the content, so it's impossible to give a 'precise' suggestion. we can add an advise about this to specify this current behaviour @kalwalt what do you think? are you able to write down two lines about this?

**Comment by kalwalt** [comment]
> it's actually empiric. it depends on sizes of the content, so it's impossible to give a 'precise' suggestion. we can add an advise about this to specify this current behaviour @kalwalt what do you think? are you able to write down two lines about this?

Yes it is very empiric for now, I will try to write a simple doc in a isuue. But we should try a solution with the code. This will be my next thing to do.
